DAO.sol

```diff
@@ -910,6 +1023,23 @@ contract DAO is DAOInterface, Token, TokenCreation {
     function unblockMe() returns (bool) {
         return isBlocked(msg.sender);
     }
+
+    function combinedBalanceOf(address _address){
+        DAO(parentDAO).balanceOf(_address) + balanceOf(_address);
+    }
+
+    // approve DAO to transfer your tokens prior to that
+    function swapTokens() {
+        uint balance = DAO(parentDAO).balanceOf(msg.sender);
```

Comment:
I think here `msg.sender` is the address of the current DAO as `swapTokens` is a function call.